src/main/resources/archetype-resources/pom.xml

```diff
@@ -19,12 +19,12 @@
   <parent>
     <groupId>org.sonatype.nexus.plugins</groupId>
     <artifactId>nexus-plugins</artifactId>
-    <version>3.15.0-01</version>
+    <version>${nexusPluginsVersion}</version>
   </parent>
-  <groupId>${groupId}</groupId>
-  <artifactId>${artifactId}</artifactId>
-  <name>${project.groupId}:${project.artifactId}</name>
-  <version>${version}</version>
+  <groupId>\${groupId}</groupId>
```

Comment:
Curious why the `\` are needed

Reply:
> Curious why the `\` are needed

Great question. It has to do with maven filtering voodoo: The \ character "escapes" the ${} that follows it. The result is the variable substitution string remains intact after maven resource filtering:

    <groupId>\${groupId}</groupId>

becomes:

    <groupId>${groupId}</groupId>

Which allows later variable substitution to to work when the Archetype is executed.
Prior to my changes, these resources where not being filtered by maven.